content/en/agent/logs/advanced_log_collection.md

```diff
@@ -453,7 +453,7 @@ spec:
 {{% /tab %}}
 {{< /tabs >}}
 
-<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em></div>
+<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with procesion of up to millisecond, so logs with greater precision will not be sent even if they match the pattern.</div>
```

Comment:
```suggestion
<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with precision of up to millisecond, so logs with greater precision are not sent even if they match the pattern.</div>
```

did you mean "procession" or "precision" here?

Reply:
Yes, that was a typo, thank you